js/app/fishwatcher.js

```diff
@@ -212,6 +212,10 @@ class FishWatcher {
     if (_(fish.predators).size() > 0) {
       var overallPredRange = null;
       var predatorsAlwaysAvailable = true;
+      // This key value is in seconds
+      var intuitionLength = fish.intuitionLength;
+      // If no key was defined, default to 1 Eorzean hour duration
+      if(!intuitionLength) intuitionLength = 175;
```

Comment:
Few things here.  Fix formatting (space after `if`, but also, since this is being used against the ranges, it actually needs to be in eorzean time.  I would use minutes in that case, so set this to 60.  Also, compare with `undefined` as: `intuitionLength !== undefined`.

If it is defined, convert the earth seconds into eorzean minutes.  Should be able to use the eorzeaTime functions for that I think.

Reply:
Minutes don't work too well because of non-aligning windows so I kept it in seconds.
Formatting fixed, added brackets as well to match what seemed the most common option throughout the code.